### PR TITLE
add repeat_interleave

### PR DESCRIPTION
**Pull request by PatternFinder**
to frontends.torch

issue: #3279

### CODE REVIEW

File: ivy/functional/frontends/torch/miscellaneous_ops.py [hunk 1]
```diff
@@ -5,19 +5,9 @@ def flip(input, dims):
     return ivy.flip(input, dims)
 
 
-flip.unsupported_dtypes = (
```
Comment (Unevilicorn) [inline ivy/functional/frontends/torch/miscellaneous_ops.py:8]: It looks like you've removed these lines by accident, please add this back in

File: ivy/functional/frontends/torch/miscellaneous_ops.py [hunk 2]
```diff
@@ -5,19 +5,9 @@ def flip(input, dims):
...
-    "uint32",
-    "uint64",
-)
-
-
 def roll(input, shifts, dims=None):
     return ivy.roll(input, shifts, dims)
 
 
-roll.unsupported_dtypes = (
```
Comment (Unevilicorn) [inline ivy/functional/frontends/torch/miscellaneous_ops.py:19]: It looks like you've removed these lines by accident, please add this back in

File: ivy/functional/frontends/torch/miscellaneous_ops.py [hunk 3]
```diff
@@ -5,19 +5,9 @@ def flip(input, dims):
...
-    "uint16",
-    "uint32",
-    "uint64",
-)
+def repeat_interleave(input, repeats, dim=None):
```
Comment (Unevilicorn) [inline ivy/functional/frontends/torch/miscellaneous_ops.py:12]: The torch `repeat_interleave` function should also take in an optional keyword argument `output_size`, you can see it [here](https://pytorch.org/docs/stable/generated/torch.repeat_interleave.html)

**Review by Unevilicorn — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/torch/miscellaneous_ops.py (hunk 3 above)
Comment (PatternFinder) [reply ivy/functional/frontends/torch/miscellaneous_ops.py]: added

File: ivy/functional/frontends/torch/miscellaneous_ops.py [hunk 4]
```diff
@@ -38,3 +38,7 @@ def cumsum(input, dim, *, dtype=None, out=None):
     "uint32",
     "uint64",
 )
+
+
+def repeat_interleave(input, repeats, dim=None, *, output_size=None):
+    return ivy.repeat(input, repeats, dim=dim, output_size)
```
Comment (Unevilicorn) [inline ivy/functional/frontends/torch/miscellaneous_ops.py:44]: This line is causing an error

**Review by Unevilicorn — CHANGES_REQUESTED** [review]
The CI is showing that the test is failing for torch for this function

**Pull request closed without merging**